### PR TITLE
Version 2.0.0

### PR DESCRIPTION
**Pull request by gakimball**
- [x] `FrontRouter` class to handle route data
- [x] API usable inside and outside of Gulp
- [x] Incremental building
- [x] Adapter system
  - [x] Generic adapter
  - [x] Angular 1.x adapter
- [x] Babel workflow (convert to ES5 before publishing)
- [ ] CLI (as separate repository/module)

### CODE REVIEW

**Comment by gakimball** [comment]
@soumak77 The Angular route creator (under `libraries/angular.js`) needs to be updated to match however the new version of the library handles routes, in terms of service naming and such.

**Comment by gakimball** [comment]
@soumak77 Also regarding [this comment](https://github.com/zurb/front-router/pull/6#issuecomment-239639462), if you need a shorter and more generic name, I'd recommend `base-router`.

**Comment by soumak77** [comment]
@gakimball I like the renaming.  Would you suggest just renaming this repository and releasing v2 as `base-router` or create a new repository and start fresh with a 1.0? No one else seems to be using this repository yet so we have the option.  The template and docs can be updated as necessary.

**Comment by soumak77** [comment]
@gakimball btw, you are welcome to make these changes yourself as a Base Apps org owner, but of course I'm willing to help out as needed.  The only requirements I have is that it works with the [angular-base-apps](https://github.com/base-apps/angular-base-apps) gulp build and the [angular-base-apps-template](https://github.com/base-apps/angular-base-apps-template) brunch build, but any other changes to make it usable outside of the framework are nice to have.

File: src/libraries/angular.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = (routes) => {
+  return `
+  angular.module('foundation.dynamicRouting').config([
```
Comment (soumak77) [inline src/libraries/angular.js:3]: `foundation.dynamicRouting` can be renamed to `dynamicRouting` and `FoundationStateProvider` can be renamed to `BaseAppsStateProvider`.  These changes are available as part of [angular-dynamic-routing](https://github.com/base-apps/angular-dynamic-routing) v1.1

**Comment by soumak77** [comment]
@gakimball should have checked this before renaming, but `base-router` is taken in npm.  I'll rename to `base-apps-router`.

**Pull request merged**